my-config/dom0-powertop.sls

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+dom0:
```

Comment:
It's cleaner to write it this way:

```
powertop:
  pkg.installed:
    - fromrepo: fedora
  service.running:
    - enable: True
```

Reply:
In fact, we want the server to be enabled, but not running. Powertop is not a service, but oneshoot. It is ran and then die. When launched how you suggest, the formula fails because the service is dead when checked. 

It works as it is.